crates/sozo/ops/src/statistics.rs

```diff
@@ -55,19 +93,30 @@ pub fn get_contract_statistics_for_dir(
         let contract_name: String =
             path.file_stem().context("Error getting file name")?.to_string_lossy().to_string();
 
+        // To ignore files like `contract.contract_class.json` or
+        // `contract.compiled_contract_class.json`
+        if contract_name.contains('.') {
```

Comment:
Those one are not usually present in a dojo project, but good to keep this in case. 👍

Reply:
it would cause issue when someone adds `sierra = true` or `casm = true` in manifest file manually for any reason, which i did for testing file size of casm class.